lib/quick_travel/booking.rb

```diff
@@ -296,6 +296,11 @@ def secure_access_code
       Encrypt.access_key(@id.to_s)
     end
 
+    def customer_comments
+      comment = comments.select{ |comment| comment['comment_type'] == 'customer' }.first
+      comment.present? ? comment['text'] : ''
```

Comment:
This assumes we're using Rails. That's a pretty good assumption. Since we're making that we can use this syntax:
```
comment.presence || ''
```
`presence` is the value or `nil` which can then work neatly with `||`.